roles/contiv_network/templates/netplugin.j2

```diff
@@ -1 +1,6 @@
-NETPLUGIN_ARGS=""
+{# If an interface is specified for netplugin, use it #}
+{% if netplugin_if is defined %}
+NETPLUGIN_ARGS='-docker-plugin -vlan-if {{netplugin_if}} -vtep-ip {{node_addr}}'
+{% else %}
+NETPLUGIN_ARGS='-docker-plugin -vtep-ip {{node_addr}}'
```

Comment:
Already opened an issue for this. https://github.com/contiv/netplugin/issues/244